Sources/MembraneRTC/Events/Event.swift

```diff
@@ -366,7 +351,7 @@ struct OfferDataEvent: ReceivableEvent, Codable {
 
 struct TracksAddedEvent: ReceivableEvent, Codable {
     struct Data: Codable {
-        let peerId: String
+        let endpointId: String
```

Comment:
I'm not sure I like, that you're sometimes using `endpointId` and sometimes `id` (eg. line 326) for no apparent reason. I'd prefer you to use only one of them (in my opinion endpointId is better so it's kept consistant with the past version where only `peerId` was used).

Reply:
I agree it would be better for this to be consistent but the backend dictates this:
https://github.com/jellyfish-dev/membrane_rtc_engine/blob/master/internal_docs/webrtc_media_events.md#endpointadded here is `id`
https://github.com/jellyfish-dev/membrane_rtc_engine/blob/master/internal_docs/webrtc_media_events.md#endpointupdated here is `endpointId`
Good catch with EndpointUpdated event though 👍